win/Projects/libsndfile/config.h

```diff
@@ -23,61 +23,10 @@ typedef unsigned __int64  uint64_t;
 #define ssize_t           SSIZE_T
 #define __func__          __FUNCTION__
 
```

Comment:
Why does the big block need to be deleted? The two lines added near the bottom will be a lot easier to get merged. This is Audacity code so doesn't need to go upstream (upstream doesn't support visual studio as a compiler).